<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HARYA-PC" name="com.test.testcase.TestUI" tests="1" failures="0" timestamp="19 Jun 2016 08:05:37 GMT" time="26.822" errors="1">
  <testcase name="testUI" time="26.822" classname="com.test.testcase.TestUI">
    <error type="java.lang.IllegalStateException" message="No match found">
      <![CDATA[java.lang.IllegalStateException: No match found
	at java.util.regex.Matcher.group(Unknown Source)
	at java.util.regex.Matcher.group(Unknown Source)
	at com.test.control.RegExp.find(RegExp.java:18)
	at com.test.control.RegExp.objectName(RegExp.java:105)
	at com.test.control.RegExp.parameterName(RegExp.java:223)
	at com.test.widget.Open.open(Open.java:41)
	at com.test.control.TestCaseDefined.selectObjectByKeyword(TestCaseDefined.java:128)
	at com.test.control.TestCaseDefined.run(TestCaseDefined.java:93)
	at com.test.control.TestCaseDefined.run(TestCaseDefined.java:78)
	at com.test.testcase.TestUI.testUI(TestUI.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testUI -->
</testsuite> <!-- com.test.testcase.TestUI -->
